src/pages/billing/List.js

```diff
@@ -200,26 +186,35 @@ export const BillingList = _.flow(
     }
   }
 
-  async componentDidMount() {
-    this.loadProjects()
-  }
+  componentDidMount = Utils.withBusyState(v => this.setState({ isBusy: v }),
+    () => Promise.all([
+      this.loadProjects(),
+      this.loadAccounts()
+    ])
+  )
 
-  loadProjects = _.flow(
-    withErrorReporting('Error loading billing projects list'),
-    Utils.withBusyState(v => this.setState({ isBusy: v }))
-  )(async () => {
-    const { ajax: { Billing } } = this.props
-    const rawBillingProjects = await Billing.listProjects()
-    const billingProjects = _.flow(
-      _.groupBy('projectName'),
-      _.map(gs => ({ ...gs[0], role: _.map('role', gs) })),
-      _.sortBy('projectName')
-    )(rawBillingProjects)
-    this.setState({ billingProjects })
-  })
+ loadProjects = withErrorReporting('Error loading billing projects list',
+   async () => {
+     const { ajax: { Billing } } = this.props
+     const rawBillingProjects = await Billing.listProjects()
+     const billingProjects = _.flow(
+       _.groupBy('projectName'),
+       _.map(gs => ({ ...gs[0], role: _.map('role', gs) })),
+       _.sortBy('projectName')
+     )(rawBillingProjects)
+     this.setState({ billingProjects })
+   })
+
+  loadAccounts = withErrorReporting('Error loading billing accounts',
+    (async () => {
+      const { ajax: { Billing } } = this.props
+      await Auth.ensureBillingScope()
```

Comment:
Moving this to the top level means that the page will ask for billing scopes immediately on navigation, right? We should check that that's acceptable.

Reply:
I spoke with Brad, looking into this further to see what can be done to allow users to continue to modify the data they area authorized to change when they are not authorized to change the billing account.

Reply:
I spoke with Brad, looking into this further to see what can be done to allow users to continue to modify the data they area authorized to change when they are not authorized to change the billing account.

Reply:
I spoke with Brad, looking into this further to see what can be done to allow users to continue to modify the data they area authorized to change when they are not authorized to change the billing account.

Reply:
I updated the logic so the authorization scope isn't requested until necessary